paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/instrumentation/ResourcesCompatTransform.kt

```diff
@@ -0,0 +1,75 @@
+package app.cash.paparazzi.gradle.instrumentation
+
+import org.jetbrains.kotlin.konan.file.File
+import org.objectweb.asm.ClassVisitor
+import org.objectweb.asm.MethodVisitor
+
+/**
+ * [ClassVisitor] that fixes a hardcoded path in ResourcesCompat.loadFont.
+ * In this method, there is a check that font files have a path that starts with res/.
+ * This is not the case in Studio. This replaces the check with one that verifies that the path contains res/.
+ *
+ * Inspired by ASM fix:
+ * https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:rendering/src/com/android/tools/rendering/classloading/ResourcesCompatTransform.kt;drc=5bb41b6d5e519c891a4cd6149234138faa28e1af
+ */
+internal class ResourcesCompatTransform(api: Int, delegate: ClassVisitor) : ClassVisitor(api, delegate) {
```

Comment:
what's different between this and https://github.com/cashapp/paparazzi/pull/679?  Would be nice to see if that version can be merged in.

Reply:
Its the same just without the platform plumbing that was implemented. I didn't want to push to that branch so I created this one.